wp-rocket.php

```diff
@@ -26,6 +26,7 @@
 define( 'WP_ROCKET_PHP_VERSION',           '7.3' );
 define( 'WP_ROCKET_PRIVATE_KEY',           false );
 define( 'WP_ROCKET_SLUG',                  'wp_rocket_settings' );
+define( 'WP_ROCKET_DEBUG_SLUG',            'wp_rocket_debug' );
```

Comment:
I think we should move away from using global constants like this. We can use the option name directly, with our architecture now it's more centralized.